netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1159,6 +1173,10 @@ net_ebpf_extension_sock_addr_authorize_connection_classify(
     _net_ebpf_extension_sock_addr_copy_wfp_connection_fields(
         incoming_fixed_values, incoming_metadata_values, &net_ebpf_sock_addr_ctx);
 
+    // eBPF programs will not be invoked on connection re-auth.
+    if (net_ebpf_sock_addr_ctx.flags & FWP_CONDITION_FLAG_IS_REAUTHORIZE)
+        goto Exit;
```

Comment:
The default action is `BPF_SOCK_ADDR_VERDICT_REJECT`. I think in re-auth case, we should change the action to `PERMIT`?

Reply:
Sorry about that. Fixed. Also filed https://github.com/microsoft/ebpf-for-windows/issues/2077, for test gap.